Add procedure, concept, and reference for image mode hosts (#3663)
theforeman/foreman-documentation: guides/common/modules/proc_viewing-booted-container-images.adoc

> [id="viewing_booted_container_images_"]

Reviewer: ```suggestion
[id="viewing-booted-container-images"]
```

= Viewing booted container images
These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.

.Procedure

> . Navigate to *Content* > *Booted Container Images* to view all images used by image mode hosts.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Booted Container Images* to view all images used by image mode hosts.
```

This page displays the spread of digests under specific image paths.
More digests under an image path indicate greater drift in the host ecosystem, ideally minimized by ensuring all hosts run the most up-to-date image version.
. Click the host count to navigate to the *All Hosts* page, which identifies the hosts associated with each specific image and displays the number of hosts using each digest.
. Select a host and on the *Details* tab, use the image mode *Details* card to see the current `bootc` status.
. Click the link to the *Bootc Action - Script Default* remote execution job in this card to perform a Bootc action on the host.

> For pull-mode remote execution users, the `foreman_ygg_worker` package must be included in the image mode host's container image.

Reviewer 2: ```suggestion
For pull-mode remote execution, the `foreman_ygg_worker` package must be included in the container image of the image mode hosts.
```

> [id="Viewing_booted_container_images_{context}"]

Reviewer 2: Please, drop `{context}` from all IDs and use lower-case-with-hyphens in IDs.

= Viewing booted container images
These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.

.Prerequisites

> * *Pull-Mode Remote Execution*: For pull-mode remote execution users, the `foreman_ygg_worker` package must be included in the image mode host's container image.

Reviewer 2: Move this sentence to the reference with REX jobs.

> * *Pull-Mode Remote Execution*: For pull-mode remote execution users, the `foreman_ygg_worker` package must be included in the image mode host's container image.

Reviewer 2: This is relevant for REX jobs, right? So it should go together with the REX jobs, right?

Additionally, I think that the `foreman_ygg_worker` package by itself is insufficient and users also need `yggdrasil`. Please, check that with a SME.

Reviewer 3: `foreman_ygg_worker` also pulls in `yggdrasil` and `foreman_ygg_migration`

Reviewer 3: Side note -- it looks like yggdrasil on image mode hosts might be broken, I'm following up with Endeavour, but we might need a known issue until it's sorted.

Author: It was confirmed from several SMEs that it's `foreman_ygg_worker`.
